utils/logger.js

```diff
@@ -0,0 +1,24 @@
+const pino = require('pino');
+// eslint-disable-next-line no-unused-vars
+const config = require('../src/config');
+
+/* If in Production mode (assumed by default):
+ * Define a new Logger instance.
+ * Write logs to a specified path.
+ * Set log level to LOG_LEVEL environment variable with 'info' as default level.
+ */
+let logger = pino({ level: process.env.LOG_LEVEL || 'info' }, pino.destination('./logFile'));
```

Comment:
Move this into an `else` below, so you don't overwrite the `logger` instance for no reason.